packages/components/addon/components/hds/table/index.hbs

```diff
@@ -4,30 +4,34 @@
   {{else if @caption}}
     <caption class="sr-only">{{@caption}}</caption>
   {{/if}}
+  <colgroup>
+    {{#if @columns}}
+      {{#each @columns as |column|}}
+        {{#if column.width}}
+          {{!template-lint-disable no-shadowed-elements}}
+          <col width={{column.width}} />
```

Comment:
[issue/blocker?] one thing I forgot to mention: with this approach, is not possible to apply a width in case of a table with no model defined (any table for which we don't provide a `columns` object).

Reply:
we changed it so you always have to provide a `columns` array hash. This brings other challenges but I scheduled a pairing session for us to discuss.